src/pages/speaker/[slug].astro

```diff
@@ -38,6 +38,12 @@ function getTwitterUsername(url: string): string | undefined {
   return (username ?? url).startsWith("@") ? username : `@${username}`;
 }
 
+// Get @username from Twitter URL
+function getBlueskyUsername(url: string): string | undefined {
+  const username = url.split("/").pop();
+  return (username ?? url).startsWith("@") ? username : `@${username}`;
```

Comment:
Good that this is being careful to avoid double @ symbols, in case the input data contains one, for example:
```json
"bluesky_url": "https://bsky.app/profile/@jimena-y-yo.bsky.social",
```
So the page correctly shows just one. For example: "Bluesky [@jimena-y-yo.bsky.social](https://bsky.app/profile/@jimena-y-yo.bsky.social)" at https://ep-website--1101.org.readthedocs.build/speaker/jimena-bermudez/

However, the link to https://bsky.app/profile/@jimena-y-yo.bsky.social is a 404. https://bsky.app/profile/jimena-y-yo.bsky.social works.

I think Bluesky URLs are not meant to contain @ signs?

Should we fix the input data and/or remove those @ signs in the render?

Reply:
Hey Hugo! Thanks for reporting this. I'll handle this case in the [programapi](https://github.com/EuroPython/programapi).

Reply:
Fixed in https://github.com/EuroPython/programapi/pull/119